kafka-commons/src/main/java/com/mesosphere/dcos/kafka/config/JmxConfig.java

```diff
@@ -0,0 +1,247 @@
+package com.mesosphere.dcos.kafka.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * JmxConfig contains the configuration for the JVM jmx props for a Kafka
+ * broker.
+ * 
+ * http://docs.oracle.com/javase/7/docs/technotes/guides/management/agent.html
+ */
+public class JmxConfig {
+    @JsonProperty("remote")
+    private boolean remote = true;
```

Comment:
the defaults are from here http://docs.oracle.com/javase/7/docs/technotes/guides/management/agent.html